e2e/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "e2e-x-dash",
+  "version": "0.0.0",
+  "description": "This module enables you to write x-dash components that respond to events and change their own data.",
+  "keywords": [
+    "x-dash"
+  ],
+  "author": "",
+  "license": "ISC",
+  "scripts": {
+    "server": "node server.mjs",
```

Comment:
**question**: what's the purpose of this? it's not used by the tests, is it only for manual testing?

Reply:
ah yes for this one, I was attempting to refactor the e2e test, so that the server runs in parallel and not in the same process as jest, to fix the issue of closing Jest when running a server 
https://stackoverflow.com/questions/51968480/jest-detects-open-handle-with-express-app